tool/tsh/tsh.go

```diff
@@ -2887,6 +2893,8 @@ func makeClientForProxy(cf *CLIConf, proxy string, useProfileLogin bool) (*clien
 		c.JumpHosts = hosts
 	}
 
+	var key *client.Key
```

Comment:
`key` needs to stay in scope so it can be used later.
https://github.com/gravitational/teleport/blob/f9ea6e4ba65cc43f05a340d462aa5ac6ec2ea30d/tool/tsh/tsh.go#L3149-L3160